doctypes/3id.md

```diff
@@ -53,29 +48,37 @@ Signed records in 3ID are stored in IPFS using the `dag-jose` or `dag-cose` form
 
 ## Update rules
 
-All valid updates must be signed by the current `management` key. An update can change anything about the content of the document except remove the `management` key. It can however change the `management` key. This is called a *key rotation*. 
+All valid updates must be signed by the current management key as listed in the `owners` array. An update can change anything about the content of the document. It can also change the management key in the `owners` array. This is called a *key rotation*.
 
-## DID document
+## DID method
+
+### Identifier
+
+The DID standard encodes DID strings differently from Ceramic identifiers. A 3ID is represented as `did:3:<genesis-CID>` in the DID standard.
+
+### DID document
 
 The [DID specification](https://w3c.github.io/did-core/) describes what a DID document should look like. This format is a bit different than how the data is stored in the 3id ceramic document. Exactly how the conversion between the two will take place is yet to be determined.
 
 ## Example
 
-The example below is the `content` of a typical 3ID document. The `collection` property contains *Collection Policies* as keys and *Privacy Policies* as values. The former is a description of a data set, usually created by an application, and the latter is the users own settings for this collection, such as specific database references and where they are hosted. In the `accountLinks` property any account-link documents that link blockchain accounts to this 3ID can be found.
+The example below is the `content` of a typical 3ID document. The `collection` property contains an array of  **Collection Tiles** and **Privacy Tiles**. The former is a description of a data set, usually created by an application, and the latter is the users own settings for a collection, such as specific database references and where they are hosted. In the `accountLinks` property any account-link documents that link blockchain accounts to this 3ID can be found.
```

Comment:
Change "Collection Tiles" and "Privacy Tiles" to: "Collection Policy" and "Privacy Policy tiles"